uint8 robot_id
string key				# This corresponds to the name of the column in the GUI, ie. "Battery(mins)", "Comms" etc..
string value		       		# The text to put in the cell
std_msgs/ColorRGBA color		# The color of the cell